README.md

```diff
@@ -48,7 +48,6 @@ Ballerina code is distributed under [Apache license 2.0](https://github.com/ball
-||[serdes](https://github.com/ballerina-platform/module-ballerina-serdes)|[![GitHub Release](https://img.shields.io/github/v/release/ballerina-platform/module-ballerina-serdes?sort=semver&color=30c955&label=)](https://github.com/ballerina-platform/module-ballerina-serdes/releases)|[![Build](https://img.shields.io/github/actions/workflow/status/ballerina-platform/module-ballerina-serdes/build-timestamped-master.yml?branch=main&label=)](https://github.com/ballerina-platform/module-ballerina-serdes/actions/workflows/build-timestamped-master.yml)|[![Trivy](https://img.shields.io/github/actions/workflow/status/ballerina-platform/module-ballerina-serdes/trivy-scan.yml?branch=main&label=)](https://github.com/ballerina-platform/module-ballerina-serdes/actions/workflows/trivy-scan.yml)|[![CodeCov](https://codecov.io/gh/ballerina-platform/module-ballerina-serdes/branch/main/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-serdes)|[![Bugs](https://img.shields.io/github/issues-search/ballerina-platform/ballerina-standard-library?query=is%3Aopen%20label%3Amodule%2Fserdes%20label%3AType%2FBug&label=&color=yellow)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aopen%20label%3Amodule%2Fserdes%20label%3AType%2FBug)|[![GitHub Pull Requests](https://img.shields.io/github/issues-pr-raw/ballerina-platform/module-ballerina-serdes.svg?label=)](https://github.com/ballerina-platform/module-ballerina-serdes/pulls)|[![Load Tests](https://img.shields.io/github/actions/workflow/status/ballerina-platform/module-ballerina-serdes/process-load-test-result.yml?branch=main&label=)](https://github.com/ballerina-platform/module-ballerina-serdes/actions/workflows/process-load-test-result.yml)|[![GraalVM Check](https://img.shields.io/github/actions/workflow/status/ballerina-platform/module-ballerina-serdes/build-with-bal-test-native.yml?branch=main&label=)](https://github.com/ballerina-platform/module-ballerina-serdes/actions/workflows/build-with-bal-test-native.yml)|
```

Comment:
No need to update this file.

Reply:
Since these changes will be auto-removed, I'll keep them for now. I'll remove them only from module_list.json in future.